src/meta/src/controller/streaming_job.rs

```diff
@@ -1330,6 +1332,20 @@ impl CatalogController {
                     }
                 });
             }
+            if is_fs_source && *fragment_type_mask == PbFragmentTypeFlag::FragmentUnspecified as i32
+            {
+                // when create table with fs connector, the fragment type is unspecified
```

Comment:
This seems still incomplete to me: I guess MV on fs source still won't work?

Reply:
It works but it breaks in another way. 

Test locally

```
dev=> CREATE source diamonds (
    carat FLOAT,
    cut TEXT,
    color TEXT,
    depth FLOAT,
) WITH (
    connector = 'posix_fs',
    match_pattern = 'data*.csv',
    posix_fs.root = 'e2e_test/source_inline/fs/data',
    source_rate_limit = 0
) FORMAT PLAIN ENCODE CSV ( without_header = 'false', delimiter = ',');
CREATE_SOURCE
dev=> create materialized view mv as select * from diamonds;
CREATE_MATERIALIZED_VIEW
dev=> select * from mv ;
 carat | cut | color | depth
-------+-----+-------+-------
(0 rows)

dev=> ALTER source  diamonds SET source_rate_limit TO DEFAULT;
ALTER_SOURCE
dev=>
dev=> flush;
FLUSH
dev=>  select * from mv ; <- still not consuming 
 carat | cut | color | depth
-------+-----+-------+-------
(0 rows)

```